src/include/version.h

```diff
@@ -14,9 +14,6 @@ extern const unsigned int coreboot_version_timestamp;
 
 /* When coreboot was compiled */
 extern const char coreboot_compile_time[];
-extern const char coreboot_compile_by[];
```

Comment:
@miczyg1 those edits are unnecessary. I know those vars are not used, but do not change them without a valid reason. Anyway this is legacy version - let's keep the changes to necessary minimum.